Sources/SourceKitLSP/Swift/SwiftTestingScanner.swift

```diff
@@ -299,6 +315,7 @@ final class SyntacticSwiftTestingTestScanner: SyntaxVisitor {
       id: (parentTypeNames + [name]).joined(separator: "/"),
       label: attributeData.displayName ?? name,
       disabled: attributeData.isDisabled || allTestsDisabled,
+      isExtension: isScanningExtension,
```

Comment:
Shouldn’t we just annotate the `TestItem` for the extension as `isExtension`, instead of also annotating the functions inside the extension?